engines/grim/textobject.cpp

```diff
@@ -214,30 +214,50 @@ void TextObject::setupTextReal(S msg, Common::String (*convert)(const S &s)) {
 	S currLine;
 	_numberLines = 1;
 	int lineWidth = 0;
+	bool isMultiByte = false;
 	for (uint i = 0; i < msg.size(); i++) {
 		message += msg[i];
 		currLine += msg[i];
-		lineWidth += _font->getCharKernedWidth(msg[i]);
+		if (i < msg.size()-1 && g_grim->getGameType() == GType_GRIM && g_grim->getGameLanguage() == Common::KO_KOR && _font->isKoreanChar(msg[i], msg[i+1])) {
```

Comment:
Please, per our code formatting conventions, write `i + 1` and `i - 1`, e.g. operators must be surrounded by spaces.